Code/src/main/java/nl/utwente/viskell/ui/CircleMenu.java

```diff
@@ -22,11 +24,23 @@
  */
 public class CircleMenu extends CirclePopupMenu {
 
+    /** List of all block on which a CircleMenu is active. */
+    private static ArrayList<Block> activeBlocks = new ArrayList<>();
```

Comment:
That's true, but "a block has-a menu" makes more sense to me than "the concept of circle menus has-a set of blocks". I'll have to think about this.